submissions/Shrai-dev/html-css-popup/index.html

```diff
@@ -0,0 +1,80 @@
+                        <li class="submenu__list-item"><a href="#"><img src="popup-icons/videos.png"
+                                    alt="Videos">Videos</a></li>
+                        <li class="submenu__list-item"><a href="#"><img src="popup-icons/search.png"
+                                    alt="Search">Search</a></li>
+                        <li class="submenu__list-item"><a href="#"><img src="popup-icons/photos.png"
+                                    alt="Photo">Photo</a></li>
+                        <li class="submenu__list-item"><a href="#"><img src="popup-icons/apps.png" alt="Apps">Apps</a>
+                        </li>
+                        <li class="submenu__list-item"><a href="#"><img src="popup-icons/documents.png"
+                                    alt="Document">Document</a></li>
+                        <li class="more__list">
+                            <input type="checkbox" class="more__popup" id="more-popup">
+                            <label class="more__submenu" for="more-popup">More</label>
+                            <ul class="more__popup-menu">
+                                <li class="submenu__list-item"><a href="#"><img src="popup-icons/chat.png"
+                                            alt="Chat">Chat</a></li>
+                                <li class="submenu__list-item"><a href="#"><img src="popup-icons/classroom.png"
+                                            alt="Classroom">Classroom</a></li>
+                                <li class="submenu__list-item"><a href="#"><img src="popup-icons/collection.png"
+                                            alt="Collection">Collection</a></li>
+                                <li class="submenu__list-item"><a href="#"><img src="popup-icons/mars.png"
+                                            alt="Mars">Mars</a></li>
+                                <li class="submenu__list-item"><a href="#"><img src="popup-icons/blogs.png"
+                                            alt="Blogs">Blogs</a></li>
+                                <li class="submenu__list-item"><a href="#"><img src="popup-icons/promo.png"
+                                            alt="Promos">Promos</a></li>
+                                <li class="more__list"><a class="more__popup-link" href="https://github.com/kottans" target="_blank">Even more from Kottans</a>
+                                </li>
+                            </ul>
+                        </li>
+                    </ul>
+                </li>
+                <li class="nav__menu-item"><a href="#"><img src="popup-icons/bell.png" alt="Bell" width="25" height="30"></a></li>
+                <li class="nav__menu-item"><a href="#"><img class="user" src="popup-icons/user.png" alt="Ico"
+                            width="30"></a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
+        <img src="" alt="">
+    </main>
```

Comment:
I think that you should add way to your `img` and add description to `alt` or delete it's block

Reply:
Done.